tests/e2e/lib/virt_storage_helpers.go

```diff
@@ -21,57 +20,10 @@ var dataVolumeGVK = schema.GroupVersionResource{
 	Version:  "v1beta1",
 }
 
-func (v *VirtOperator) deletePvc(namespace, name string) error {
-	return v.Clientset.CoreV1().PersistentVolumeClaims(namespace).Delete(context.Background(), name, metav1.DeleteOptions{})
-}
-
-func (v *VirtOperator) detachPvc(namespace, name string) error {
-	pvc, err := v.Clientset.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), name, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-	if pvc == nil {
-		return fmt.Errorf("PVC %s/%s does not exist", namespace, name)
-	}
-
-	owners := make([]metav1.OwnerReference, 0)
-	for _, owner := range pvc.OwnerReferences {
-		if owner.Kind == "DataVolume" {
-			continue
-		}
-		owners = append(owners, owner)
-	}
-
-	pvc.OwnerReferences = owners
-	_, err = v.Clientset.CoreV1().PersistentVolumeClaims(namespace).Update(context.Background(), pvc, metav1.UpdateOptions{})
-	return err
-}
-
-func (v *VirtOperator) checkPvcExists(namespace, name string) bool {
-	pvc, err := v.Clientset.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), name, metav1.GetOptions{})
-	if err != nil {
-		return false
-	}
-	if pvc == nil {
-		return false
-	}
-	return true
-}
-
-// Check if this PVC is still owned by a DataVolume.
-func (v *VirtOperator) checkPvcAttached(namespace, name string) bool {
-	pvc, err := v.Clientset.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), name, metav1.GetOptions{})
-	if err != nil || pvc == nil {
-		return false
-	}
-
-	for _, owner := range pvc.OwnerReferences {
-		if owner.Kind == "DataVolume" {
-			return true
-		}
-	}
-
-	return false
+var dataSourceGVK = schema.GroupVersionResource{
```

Comment:
here should be GVR? (Resource instead of Kind) (this may also need fixing in `virt_helpers.go` for standardization)

Reply:
Done.